publication-commons/src/main/java/no/unit/nva/publication/JsonLdContextUtil.java

```diff
@@ -2,13 +2,14 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import java.io.InputStream;
-import java.nio.file.Path;
-import java.util.Optional;
 import nva.commons.utils.IoUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.InputStream;
```

Comment:
run an "Organize imports" on the whole project

Reply:
I did.